usagov/vote-gov-drupal
# Feature/vote 938/refine styles banner

## Jira ticket
[VOTE-990](https://cm-jira.usa.gov/browse/VOTE-990)
[VOTE-938](https://cm-jira.usa.gov/browse/VOTE-938)

## Description

Refinements for the language selector and banner styles.

## Deployment and testing

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Insert any required steps to take before beginning test.

### QA/Test

1. lando retune
2. npm run build

[VOTE-990](https://cm-jira.usa.gov/browse/VOTE-990)
4. to test language selector you may need to add a translation to be sure there are 2 so the lang selector is available. 
5. check that the lang selector button position looks correct. 
6. add more languages so there are more than 2 and confirm again.

[VOTE-938](https://cm-jira.usa.gov/browse/VOTE-938)
7. Compare banner height to figma design and refinement notes in ticket.

## Checklist for the Developer




## Checklist for the Peer Reviewers
